<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministrationBillet</class>
 <widget class="QDialog" name="AdministrationBillet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administration de billet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1062</width>
          <height>725</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QGroupBox" name="gBoxModePaiement">
               <property name="font">
                <font>
                 <family>Verdana</family>
                 <pointsize>12</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="title">
                <string>Modes de paiement</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QTableView" name="TabVMPaiement"/>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QLabel" name="AdBLabelMPaiement">
                      <property name="text">
                       <string>Mode de paiement</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="AdBTxtIntituleMPaiement"/>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_9"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPushButton" name="AdBBtnModifierFicheMPaiement">
                    <property name="text">
                     <string>Modifier la fiche</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_10">
                    <item>
                     <widget class="QPushButton" name="AdBBtnAjouterMPaiement">
                      <property name="text">
                       <string>Ajouter </string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="AdBBtnSupprimerMPaiement">
                      <property name="text">
                       <string>Supprimer</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPushButton" name="AdBBtnAnnulerMPaiement">
                    <property name="text">
                     <string>Annuler</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="gBoxTarif">
               <property name="font">
                <font>
                 <family>Verdana</family>
                 <pointsize>12</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="title">
                <string>Tarifs</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QTableView" name="TabVTarif"/>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="AdBLabelTarif">
                      <property name="text">
                       <string>Intitulé</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="AdBTxtTarif"/>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <item>
                       <widget class="QLabel" name="AdBLabelPrix">
                        <property name="text">
                         <string>Prix</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="AdBTxtPrix"/>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPushButton" name="AdBBtnModifierFicheTarif">
                    <property name="text">
                     <string>Modifier la fiche</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QPushButton" name="AdBBtnAjouterTarif">
                      <property name="text">
                       <string>Ajouter </string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="AdBBtnModifierTarif">
                      <property name="text">
                       <string>Modifier</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="AdBBtnSupprimerTarif">
                      <property name="text">
                       <string>Supprimer</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPushButton" name="AdBBtnAnnulerTarif">
                    <property name="text">
                     <string>Annuler</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="gBoxConfigSalle">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="title">
              <string>Configurations salles</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QTableView" name="TabVConfigSalle"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QLabel" name="AdBLabelIntitule">
                    <property name="text">
                     <string>Intitulé</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="AdBTxtIntitule"/>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLabel" name="AdBLabelJauge">
                      <property name="text">
                       <string>Jauge</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="AdBTxtNbPlaces"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="AdBBtnModifierFiche">
                  <property name="text">
                   <string>Modifier la fiche</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QPushButton" name="AdBBtnAjouter">
                    <property name="text">
                     <string>Ajouter </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="AdBBtnModifier">
                    <property name="text">
                     <string>Modifier</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="AdBBtnSupprimer">
                    <property name="text">
                     <string>Supprimer</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="AdBBtnAnnuler">
                  <property name="text">
                   <string>Annuler</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="AdBtnQuitter">
         <property name="text">
          <string>Quitter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAlert">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
